pkg/helm/client.go

```diff
@@ -85,8 +85,29 @@ func (h *Client) InstallRelease(chstr, ns string, opts ...InstallOption) (*rls.I
 	return h.InstallReleaseFromChart(chart, ns, opts...)
 }
 
+// InstallReleaseWithContext loads a chart from chstr, installs it, and returns the release response while accepting a context.
+func (h *Client) InstallReleaseWithContext(ctx context.Context, chstr, ns string, opts ...InstallOption) (*rls.InstallReleaseResponse, error) {
+	// load the chart to install
+	chart, err := chartutil.Load(chstr)
+	if err != nil {
+		return nil, err
+	}
+
+	return h.installReleaseFromChartWithContext(ctx, chart, ns, opts...)
+}
+
 // InstallReleaseFromChart installs a new chart and returns the release response.
 func (h *Client) InstallReleaseFromChart(chart *chart.Chart, ns string, opts ...InstallOption) (*rls.InstallReleaseResponse, error) {
+	return h.installReleaseFromChartWithContext(context.Background(), chart, ns, opts...)
+}
+
+// InstallReleaseFromChartWithContext installs a new chart and returns the release response while accepting a context.
+func (h *Client) InstallReleaseFromChartWithContext(ctx context.Context, chart *chart.Chart, ns string, opts ...InstallOption) (*rls.InstallReleaseResponse, error) {
+	return h.installReleaseFromChartWithContext(ctx, chart, ns, opts...)
+}
+
+// InstallReleaseFromChart installs a new chart and returns the release response while accepting a context.
```

Comment:
This comment needs to be changed to match the new function signature below

Reply:
Done